osu.Game.Rulesets.Osu/Mods/Objects/StrictTrackingDrawableSliderTail.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Rulesets.Osu.Objects.Drawables;
+
+namespace osu.Game.Rulesets.Osu.Mods.Objects
+{
+    public class StrictTrackingDrawableSliderTail : DrawableSliderTail
```

Comment:
`StrictTrackingDrawableSliderTail`, `StrictTrackingSlider` and `StrictTrackingSliderTailCircle` should not be in separate files in the `osu.Game.Rulesets.Osu.Mods.Objects` namespace. They should be private/protected nested classes inside the `OsuModStrictTracking` class, because they are specific to this one mod and do not generally make sense anywhere else.